app/services/katello/pulp3/repository/file.rb

```diff
@@ -46,6 +46,10 @@ def delete_publisher(href = repository_reference.publisher_href)
         def create_publication
           pulp3_api.publishers_file_file_publish(repository_reference.publisher_href, repository_version: repo.version_href)
         end
+
+        def delete_publication
+          pulp3_api.publications_delete
+        end
```

Comment:
I don't see this delete publication called from anywhere, did you mean to leave it?  I will add a task to our list to handle cleaning up old publications.

Reply:
Removed it..Unrelated to this, somehow the VCR test is failing on the PR..Waiting for green build..